src/add/operator/buffer.ts

```diff
@@ -4,4 +4,8 @@ import {buffer} from '../../operator/buffer';
 
 Observable.prototype.buffer = buffer;
 
-export var _void: void;
\ No newline at end of file
+declare module '../../Observable' {
+  interface Observable<T> {
+    buffer: (closingNotifier: Observable<any>) => Observable<T[]>;
+  }
+}
```

Comment:
It makes more sense to me to declare an interface first and then implement it. But this is not a big deal. I'm just commenting to see if this makes sense to someone else other than me. ;)